lib/src/collect.dart

```diff
@@ -497,3 +503,14 @@ class StdoutLog extends Log {
   @override
   void severe(String message) => print(message);
 }
+
+extension _ScopedOutput on Set<String> {
+  bool includesScript(String? scriptUriString) {
+    if (scriptUriString == null) return false;
+    if (isEmpty) return true;
+    final scriptUri = Uri.parse(scriptUriString);
+    if (scriptUri.scheme != 'package') return false;
+    final scope = scriptUri.path.split('/').first;
```

Comment:
Nit: would `final scope = scriptUri.pathSegments.first` achieve the same thing?

Reply:
Done